docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+```
+expo-protocol-version: 0
+expo-sfv-version: 0
+expo-manifest-filters: expo-sfv
+expo-server-defined-headers: expo-sfv
+cache-control: *
+content-type: application/json; charset=utf-8
+```
+
+* `expo-protocol-version` describes the version of the protocol defined in this spec and MUST be `0`.
+* `expo-sfv-version`  MUST be `0`.
+* `expo-manifest-filters` is an [Expo SFV 0](expo-sfv-0.md) dictionary. It is used to filter updates stored by the client-library by the `updateMetadata` attributes found in the [manifest](#body).
+  * For example: `expo-manifest-filters: branchname="main"` instructs the client-library to load the most recent update it has stored whose `updateMetadata` contains:
+
+  ```
+  updateMetadata: {
+    branchname: 'main',
+    ...
+  }
+  ```
+  * If the `branchname` manifest filter is included, it MUST equal the `branchName` in the `manifest.updateMetadata`.
+* `expo-server-defined-headers` is an [Expo SFV](expo-sfv.md) dictionary. It defines headers that a client-library MUST store and include in every subsequent [manifest request](#manifest-request).
+
+  * For example, when rolling out an update, we require a client-library to send back a stable token: `expo-server-defined-headers: expo-rollout-token="token"`. 
+* `cache-control` We recommend `cache-control: private, max-age=0`. This allows ensures the newest manifest is returned. Setting longer cache ages could result in stale assets for users.
+
+
+The manifest endpoint MUST also be served with a `cache-control` header set to an appropriately short period of time. For example:
+
+```
+cache-control: max-age=0, private
+```
+
+### Body
+
+The body of the response MUST be a manifest. A manifest is defined as JSON conforming to the following description:
+
+The response body is referred to as the manifest and MUST be a JSON with format:
+```
+{
+  id: string
+  createdAt: datetime
+  runtimeVersion: string
+  launchAsset: Asset
+  assets: Asset[]
+  updateMetadata: { [key: string]: number | string }
+}
+```
+Where an _Asset_ is the JSON:
+```
+{
+  hash: string
+  key: string
+  contentType: string
+  url: string
+}
+```
+  * `id` The ID MUST uniquely specify the manifest.
+  * `createdAt` The date and time created is essential as the client-library selects the most recent update (subject to any constraints supplied by the `expo-manifest-filters` header). The datetime should be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+  * `runtimeVersion` Can be any string defined by the developer. It stipulates what native code setup is required to run the associated JavaScript update bundle.
+  * `Asset` Provides information about an asset and where to obtain it.
+    * `hash` SHA256 hash of the file to guarantee integrity.
+    * `key` Key generated by the bundler and used to identify this asset.
+    * `contentType` The mime type of the asset as defined by [RFC 2045](https://tools.ietf.org/html/rfc2045). e.g. `application/javascript`, `image/jpeg`.
```

Comment:
```suggestion
    * `contentType` The mime type of the file as defined by [RFC 2045](https://tools.ietf.org/html/rfc2045). e.g. `application/javascript`, `image/jpeg`.
```

(since it is referred to as a "file" in the `hash` section)

Reply:
mime -> MIME